curriculum/challenges/english/16-the-odin-project/top-learn-block-and-inline/learn-block-and-inline-question-a.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 6542212650bc5c6ed8abf746
+title: Learn Block and Inline Question A
+challengeType: 15
+dashedName: learn-block-and-inline-question-a
+---
+# --description--
+
+Most of the elements that you have learned about so far are block elements. In other words, their default style is `display: block`. By default, block elements will appear on the page stacked atop each other, each new element starting on a new line.
+
+Inline elements, however, do not start on a new line. They appear in line with whatever elements they are placed beside. A clear example of an inline element is a link, or `<a>` tag. If you stick one of these in the middle of a paragraph of text, it will behave like a part of the paragraph. [Like this…](https://www.youtube.com/watch?v=dQw4w9WgXcQ) The link’s text will sit alongside other words in that paragraph. Additionally, padding and margin behave differently on inline elements. In general, you do not want to try to put extra padding or margin on inline elements.
+
+# The middle ground inline-block
+Inline-block elements behave like inline elements, but with block-style padding and margin. `display: inline-block` is a useful tool to know about, but in practice, you’ll probably end up reaching for flexbox more often if you’re trying to line up a bunch of boxes. Flexbox will be covered in-depth in the next lesson.
```

Comment:
Small things:
1) We should not have another `h1` on our page
2) No funny characters `'` vs `’`

```suggestion
## The middle ground inline-block
Inline-block elements behave like inline elements, but with block-style padding and margin. `display: inline-block` is a useful tool to know about, but in practice, you'll probably end up reaching for flexbox more often if you're trying to line up a bunch of boxes. Flexbox will be covered in-depth in the next lesson.
```